src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/SpecifyStringComparisonTests.cs

```diff
@@ -79,8 +80,25 @@ public bool StringWith()
                                  "string.StartsWith(string, System.StringComparison)"));
         }
 
+        [Fact, WorkItem(2581, "https://github.com/dotnet/roslyn-analyzers/issues/2581")]
+        public async Task CA1307_StringWithCharTests_CSharp()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+using System;
+using System.Globalization;
+
+public class StringComparisonTests
+{
+    public bool StringWith(string strA, char chA, char chB)
+    {
+        var x = strA.EndsWith(chA);
```

Comment:
Oh, the `EndsWith(char)` and `StartsWith(char)` overloads don't exist in .NET Framework 4.7.2.

Reply:
Ah thanks, do we know if we have a way to skip tests for a specific target framework?

Reply:
#if NETCOREAPP 😄 ?

Reply:
Thanks!